task/task-5.py

```diff
@@ -0,0 +1,9 @@
+from utils.reader import read_file
+path = read_file("/home/zia/ISDP/Hammad sab/Python/wether_Mar/weather task 2 in csv file/files/f2.csv")
```

Comment:
path is not a good variable name, zia. these are file values or file contents. i expect better names from you

Reply:
also update t1 and t2 as they use the same function